pytorch_cnn_trainer/plotter.py

```diff
@@ -1,12 +1,15 @@
 import matplotlib.pyplot as plt 
 
+__all__ = ['plot']
+
 def plot(history,metric1,metric2):
-    plt.plot(history["train"][metric1])
-    plt.plot(history["train"][metric2])
-    plt.plot(history["val"][metric1])
-    plt.plot(history["val"][metric2])
+    train_m1 = plt.plot(history["train"][metric1])
+    train_m2 = plt.plot(history["train"][metric2])
+    val_m1 = plt.plot(history["val"][metric1])
+    val_m2 = plt.plot(history["val"][metric2])
     plt.title('train v/s val')
     plt.ylabel('metrics')
     plt.xlabel('epoch')
     plt.legend(['train-'+ metric1,'train-'+metric2, 'val-'+metric1,'val-'+metric2], loc='upper left')
     plt.show()
+    return [train_m1,train_m2,val_m1,val_m2]
```

Comment:
I think I have returned the object incorrectly

Reply:
Yes. Let's return None to this ;) 

Write these down in tests, rest looks fine.

Reply:
Can you just give a hint what did you actually expect as a returned object and hope it does'nt hardcoded now

Reply:
> Write these down in tests, rest looks fine.

I have written test for it